tests/unit/getTokenNames.test.ts

```diff
@@ -180,7 +180,8 @@ describe("getTokenNodes", () => {
         paddingTop: 0,
         paddingRight: 0,
         paddingBottom: 0,
-        paddingLeft: 0
+        paddingLeft: 0,
+        opacity: undefined
```

Comment:
Can this value actually be undefined? Does it not have to be between 0 and 1?

Reply:
It's undefined because the pages used in the test have no data mocked for opacity, although real page data from Figma will contain all that info.
I've modified the `extractTokenNodeValues` function to default like Figma to `1` in case there's no opacity (unlikely because all nodes have it). On top of that, I've updated test data with 2 opacities and left 1 without to use the fallback.